pkg/client/client.go

```diff
@@ -382,8 +382,13 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, sd StatusDetai
 	}
 
 	if sr.ClusterMesh != nil {
+<<<<<<< HEAD
 		fmt.Fprintf(w, "ClusterMesh:\t%d/%d clusters ready, %d global-services\n",
 			numReadyClusters(sr.ClusterMesh), len(sr.ClusterMesh.Clusters), sr.ClusterMesh.NumGlobalServices)
+=======
+		fmt.Fprintf(w, "ClusterMesh:\t%d/%d remote clusters ready, %d global-services\n",
+			NumReadyClusters(sr.ClusterMesh.Clusters), len(sr.ClusterMesh.Clusters), sr.ClusterMesh.NumGlobalServices)
+>>>>>>> 2c68c61947 (cilium-dbg: minor clarifications to the clustermesh status output)
```

Comment:
Something went wrong in the conflict resolution here. Should additionally change `failures` to `reconnections` a couple of lines below.

Reply:
Should be fixed now.